.github/workflows/test.yaml

```diff
@@ -0,0 +1,9 @@
+name: Tests
+
+on:
+  pull_request:
+
+jobs:
+  unit-tests:
+    uses: canonical/operator-workflows/.github/workflows/test.yaml@debug/yanks
+    secrets: inherit
```

Comment:
what about using the edge self-hosted runners? ;-)

Reply:
Let me add them on the follow up [PR](https://github.com/canonical/tmate-ssh-server-operator/pull/5) Thank you!